File: fe_public/quadra_mall_web-main/src/types/Auth/base/Response/loginResponse.ts
import type {Role} from "@/types/Auth/base/User";

export interface LoginResponse {
    token: string;
    refreshToken: string;
    userId: string;
    provider: string;
    email: string;
    fullName: string;
    phoneNumber?: string; 
    avatarUrl?: string;
    roles:Role[]
}

File: be/quadramall_api-main/src/main/java/com/quadra/ecommerce_api/dto/custom/auth/response/LoginResponse.java
package com.quadra.ecommerce_api.dto.custom.auth.response;

import com.quadra.ecommerce_api.dto.base.user.RoleDTO;
import com.quadra.ecommerce_api.entity.user.Role;
import jakarta.validation.constraints.NotBlank;
import lombok.*;

import java.util.List;
import java.util.Set;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class LoginResponse {

    private String token;
    private String refreshToken;
    private String provider;
    private String email;
    private String fullName;
    private String phone;
    private String avatarUrl;
    private Set<RoleDTO> roles;
    private List<Long> storeIds; // Thêm trường này
    private Long userId;


}
